contrib/babelfishpg_tsql/sql/information_schema_tsql.sql

```diff
@@ -717,3 +717,28 @@ CREATE OR REPLACE VIEW information_schema_tsql.routines AS
 GRANT SELECT ON information_schema_tsql.routines TO PUBLIC;
 
 SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false);
+
+
+CREATE OR REPLACE VIEW information_schema_tsql.domain_constraints
+AS SELECT sys.db_name()::information_schema.sql_identifier AS constraint_catalog,
+    rs.nspname::information_schema.sql_identifier AS constraint_schema,
+    con.conname::information_schema.sql_identifier AS constraint_name,
+    sys.db_name()::information_schema.sql_identifier AS domain_catalog,
+    sys.schema_name(n."oid")::information_schema.sql_identifier AS domain_schema,
+    t.typname::information_schema.sql_identifier AS domain_name,
+        CASE
+            WHEN con.condeferrable THEN 'YES'::text
+            ELSE 'NO'::text
+        END::information_schema.yes_or_no AS is_deferrable,
+        CASE
+            WHEN con.condeferred THEN 'YES'::text
+            ELSE 'NO'::text
+        END::information_schema.yes_or_no AS initially_deferred
+   FROM pg_catalog.pg_namespace rs,
```

Comment:
Use join of pg_namespace_ext and babelfish_namespace_ext instead of pg_namespace + nspname query